src/api/spec/models/workflow/step_spec.rb

```diff
@@ -62,6 +62,10 @@
   describe '#target_project_name' do
     let(:step) do
       Class.new(described_class) do
+        def self.name
```

Comment:
This is needed to make tests pass now that we have instrumentation for `Workflow::Step`. Without this, `self.class.name` in the instrumentation code returns `nil` since we're defining an anonymous class here. It's only an issue in this spec.